web/beacon-app/src/features/topics/components/TopicQuery.tsx

```diff
@@ -19,22 +19,22 @@ const TopicQuery = () => {
 
       {open && (
         <div className="flex space-x-1">
-          <Trans>
-            <p>
+          <p>
+            <Trans>
               Coming soon! Query the topic for insights with{' '}
               <span>
                 <a
                   href="#"
                   target="_blank"
                   rel="noreferrer"
-                  className="font-bold text-[#1F4CED] underline hover:!underline"
+                  className="font-bold text-[#1D65A6] underline hover:!underline"
```

Comment:
Changed this color  to match this : https://www.figma.com/file/PvL23fm6W36Zd88O1XVyIo/Beacon-DX-UI?node-id=7569%3A15483&mode=dev